lib/buildenv/builder.pl

```diff
@@ -0,0 +1,303 @@
+#! @perl@ -w
```

Comment:
Should this have a similar comment as the buildenv default.nix?

Reply:
This is a copy of the builder.pl from nixpkgs upstream, and we're looking to minimize the changes so that we can diff them over time to incorporate future changes from upstream, so no we should restrict all changes to just the necessary ones.